ovs/extensions/generic/watcher.py

```diff
@@ -67,18 +67,16 @@ def services_running(self, target):
                     self.log_message(target, '  Error during configuration store test: {0}'.format(ex), 2)
                     return False
 
+                from ovs.extensions.db.arakoon.arakooninstaller import ArakoonInstaller, ArakoonClusterConfig
                 from ovs.extensions.db.arakoon.configuration import ArakoonConfiguration
-                from ovs.extensions.db.arakoon.ArakoonInstaller import ArakoonInstaller, ArakoonClusterConfig
                 from ovs.extensions.db.arakoon.pyrakoon.pyrakoon.compat import NoGuarantee
-                with open(ArakoonConfiguration.CACC_LOCATION) as config_file:
-                    contents = config_file.read()
-                config = ArakoonClusterConfig(cluster_id='cacc', load_config=False)
-                config.read_config(contents)
+
+                config = ArakoonClusterConfig(cluster_id='cacc', source_ip='127.0.0.1')
```

Comment:
I think the `127.0.0.1` only works if you don't switch users. Might be the case here, but I don't like this "requirement".